inc/change.class.php

```diff
@@ -1163,4 +1210,39 @@ static function showListForItem(CommonDBTM $item) {
    function showDebug() {
       NotificationEvent::debugEvent($this);
    }
+
+
+   /**
+    * Summary of getTicketActors
+    * Get the list of actors for the current ticket
+    * will return an assoc array of users_id => array of roles.
+    * @return array[] of array[] of users and roles
+    * @since 0.90
+    * @deprecated
+   **/
+   function getITILActors() {
```

Comment:
I do not understrand why to add this method and deprecate it...

Reply:
Fixed. I also changed it so getTicketActors calls getITILActors instead of the other way around. It should make it a little easier to remove in the future.